server/config_check_test.go

```diff
@@ -1110,7 +1110,7 @@ func TestConfigCheck(t *testing.T) {
 			config: `
 				lame_duck_grace_period: abc
 			`,
-			err:       errors.New(`error parsing lame_duck_grace_period: time: invalid duration abc`),
+			err:       errors.New(`error parsing lame_duck_grace_period: time: invalid duration`),
```

Comment:
In Go 1.15 time.Parse errors are now within quotes, so changed to match a substring only